### ISSUE TITLE
basic algebra on computed fields

### ISSUE THREAD

**Issue by KrisThielemans** [microsoft/yardl#73]
It'd be nice to be able to do some basic manipulations for a computed field, e.g. subtracting 1
```
ScannerInformation: !record
  fields:
    # edge information for TOF bins in mm (e.g. start,edge1, ... end)
    tofBinEdges: float*
  computedFields:
    numberOfTOFBins: size(tofBinEdges)-1
```

**Comment by johnstairs** [comment]
Agree!

**Comment by johnstairs** [comment]
@KrisThielemans, this is now available in the [v0.3.1](https://github.com/microsoft/yardl/releases/tag/v0.3.1) release.
